### PR TITLE
Allow easier customizations

### PR DESCRIPTION
**Pull request by ale-rt**
The class for the `@@images` view has this definition `scale_view = ImageScale(...)` in many places.
This patch allows to customize the behavior of `ImageScale` just by setting a class attribute instead of overriding all the methods that use this class.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -10,7 +10,9 @@ Breaking changes:
 
 New features:
 
-- Target Zope 4 (test chnages only).
+- It is now possible to customize in an easier way the ``@@images`` view
```
Comment (pbauer) [inline CHANGES.rst]: can you please add a example to the README how and why to customize that?

**Comment by jensens** [comment]
Hence there was no activity for a long time, I propose to close this PR within next two weeks. If you do not feel ok with this, please speak up and provide us a rough schedule.

**Comment by ale-rt** [comment]
Well I tried hard to came up with something useful to add to the documentation.

The truth is that it is hard to do it because the `@@images` view uses internally another view (which is not registered in any zcml) and all feels wrong.

In the project I am using this branch I am customizing the `@@images` view like this in order to be able to use it on comments:
```
class CommentImageScale(ImageScale):
    ''' Customization to overcome the traversing limitation of comment objects
    '''

    def validate_access(self):
        ...


class CommentImageScaling(ImageScaling):
    ''' Customize the @@images view for the comment content type
    '''
    _scale_view_class = CommentImageScale
```

Without that patch (which is very trivial), instead of customizing one single attribute I would have to copy a lot of code just to make for replacements.

Long story short... It would really be better to fix the `ImageScaling` class to not use another unregistered browser view, but in the meanwhile this is a sort of MVP for easier customizations.

**Comment by ale-rt** [comment]
I opened another issue about the problems of the `@@images` view: https://github.com/plone/plone.namedfile/issues/79
